site/content/blog/2021-02-01-whats-new-in-svelte-february-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+title: What's new in Svelte: February 2021
+description: A Svelte-packed showcase to kick-off the new year!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With the shortest month of the year coming up, Svelte maintainers and community members alike have been busy this last month – from new aria roles in the compiler to semantic highlighting hooks in the langugage-tools. Meanwhile, lots of folks have been busy integrating Svelte with other popular frameworks.
+
+## New compiler features
+- Aria roles from the [WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/#role_definitions) are now recognized as valid aria roles in Svelte components (**3.31.1**)
+- Compiler warnings for the common React attributes, `className` and `htmlFor`, now make it easier to port React components to Svelte (**3.31.1**)
+
+Have a suggestion for a compiler feature or want to help implement new features/bug fixes? Check out the ["triage: good first issue" tag for Svelte](https://github.com/sveltejs/svelte/issues?q=is%3Aopen+is%3Aissue+label%3A%22triage%3A+good+first+issue%22)
+
+
+## New bits in language-tools
+
+- User disabled auto import suggestions no longer show in VS Code (**103.0.0**)
+- Renaming a variable is now safer with smart additions of a prefix/suffix to renamed variables (**104.0.0**)
+- Semantic (token) highlighting for typescript users lets theme makers apply Svelte-specific styling in their themes (**104.0.0**)
```

Comment:
It's not Svelte-specific styling, it's more general a TypeScript-semantic styling .

```suggestion
- Semantic (token) highlighting for TypeScript users lets theme makers apply semantic styling in their themes, if they support it (**104.0.0**)
```